src/kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -294,7 +294,17 @@ impl<'tcx> GotocCtx<'tcx> {
                 let tp_ty = instance.substs.type_at(0);
                 let arg = fargs.remove(0);
                 let size_align = self.size_and_align_of_dst(tp_ty, arg);
-                self.codegen_expr_to_place(p, size_align.$which)
+                if size_align.is_some() {
```

Comment:
`if let Some` instead of using `unwrap`?

Reply:
Yes, because this can be None.